src/java/fr/paris/lutece/util/signrequest/IPAuthentificator.java

```diff
@@ -0,0 +1,72 @@
+package fr.paris.lutece.util.signrequest;
+
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class IPAuthentificator implements RequestAuthenticator {
+	
+	private List<String> _listIPs;
+	private boolean isWhiteList;
+	
+	/**
+	 * Gets the list of IP adresses
+	 * 
+	 * @return the list of IP adresses
+	 */
+	public List<String> getIPs( ) {
+		return _listIPs;
+	}
+	
+	/**
+	 * Sets the list of IP adresses
+	 * 
+	 * @param list
+	 * 		The list of IP adresses
+	 */
+	public void setIPs( List<String> list ) {
+		this._listIPs = list;
+	}
+	
+	/**
+	 * Gets the boolean that defines if it is a white or black list
+	 * 
+	 * @return the boolean
+	 */
+	public boolean isWhiteList( ) {
```

Comment:
Using block list / allow list may be preferable to black/white list (see https://www.ncsc.gov.uk/blog-post/terminology-its-not-black-and-white for instance)